src/main/java/com/aizistral/nochatreports/config/ModMenuIntegration.java

```diff
@@ -71,6 +72,27 @@ public ConfigScreenFactory<?> getModConfigScreenFactory() {
 							.withHoverEvent(new HoverEvent(HoverEvent.Action.SHOW_TEXT, Component.translatable("https://github.com/Aizistral-Studios/No-Chat-Reports/wiki/Configuration-Files/#option-showserversafety"))).withClickEvent(new ClickEvent(ClickEvent.Action.OPEN_URL, "https://github.com/Aizistral-Studios/No-Chat-Reports/wiki/Configuration-Files/#option-showserversafety"))))
 					.build());
 
+			// Set an option for verifiedIconEnabled
+			visual.addEntry(entryBuilder.startBooleanToggle(Component.translatable("option.NoChatReports.verifiedIconEnabled"), NCRConfig.getClient().verifiedIconEnabled)
+					.setDefaultValue(true)
+					.setTooltip(this.makeTooltip("option.NoChatReports.verifiedIconEnabled.tooltip"))
+					.setSaveConsumer(newValue -> NCRConfig.getClient().verifiedIconEnabled = newValue)
+					.build());
+
+			// Set an option for verifiedIconOffsetX
+			visual.addEntry(entryBuilder.startIntSlider(Component.translatable("option.NoChatReports.verifiedIconOffsetX"), NCRConfig.getClient().verifiedIconOffsetX, -5, 5)
```

Comment:
Why are offsets limited to 5 pixels?

Reply:
Going over 5 had it work half of the time, with the ping check fix I will retest